src/_guides/bump-sh-tutorials/documenting-multiple-api-versions.md

```diff
@@ -88,23 +88,29 @@ jobs:
 ### Example: One Git branch per API version
 
 This setup involves one GitHub repository. Each version of the API is in its own
-Git branch, a `v2` and `v3`. In this instance, you could use GitHub Actions,
-with different actions in each branch.
+Git branch, a `v2` and `v3`. In this instance, you could use a GitHub Action, that gets triggered on each of the corresponding branch.
 
-In the `v2` Git branch:
+The workflow file would look like this in both Git branches:
 
 ```yaml
 # .github/workflows/bump.yml
-name: Deploy API documentation
+name: Deploy API documentation pushes
 on:
   push:
-    branches: [v2]
+    branches: [v2, v3]
 
 jobs:
   deploy-doc:
-    if: ${{ github.event_name == 'push' }}
     name: Deploy API documentation on Bump.sh
     runs-on: ubuntu-latest
+    env:
+      TARGET_BUMP_BRANCH: >
```

Comment:
Wow, very smart suggestion, I didn't know we could use such Json logic in GitHub action scripts.